content/api/tracking-domains.apib

```diff
@@ -354,10 +354,15 @@ Delete an existing tracking domain.
 
 
 
-### List All Tracking Domains [GET /v1/tracking-domains]
+### List All Tracking Domains [GET /v1/tracking-domains{?default,subaccounts,delimiter}]
 
 Returns an array with all tracking domains. Each item in the array is a full tracking domain object.
 
++ Parameters
+    + default = `false` (optional, boolean) ... Restrict results to default domains when true.
+    + subaccounts (optional, list) ... Delimited list of subaccounts to include in results.
```

Comment:
Presumably a subaccount can only see Tracking Domains created for that subaccount? So if you're requesting with an API key for subaccount X and you request subaccount Y's Tracking Domains in the subaccounts= parameter, you get an empty list?

Reply:
Correct